projects/wave-core/src/lib/operators/dialogs/data-repository/vector/vector-source-dataset.component.ts

```diff
@@ -33,7 +33,7 @@ export class VectorSourceDatasetComponent implements OnInit {
     _showPreview = false;
     _showDescription = false;
     _tableSource: LayerTableDataSource;
-    _displayedColumns  = ['title'];
+    _displayedColumns  = ['title', 'start', 'end'];
```

Comment:
Same here.

Reply:
fixed in bc0236c100809397960ca46b0f1e76b973d72473